getMarks/getMarks.js

```diff
@@ -32,35 +30,17 @@ function getMarksBySubjectId(id) {
   };
 
   if (id > 0 && id <= constants.CONSTANTS.MOCK_SUBJECTS.length) {
+    const marks = constants.CONSTANTS.MOCK_MARKS[id - 1].Marks;
+    const average = marks.reduce((prev, curr) => prev + curr) / marks.length;
```

Comment:
Use separate method for average mark value calculation, please to avoid code duplication (We need to get average mark value in different places)

Reply:
fixed